cloud/aws/alb/variables.tf

```diff
@@ -162,22 +168,16 @@ variable "latency_notifications_critical" {
   default     = []
 }
 
-variable "latency_aggregation_function" {
-  description = "Aggregation function and group by for latency detector (i.e. \".mean(by=['host'])\")"
-  type        = string
-  default     = ""
-}
-
-variable "latency_transformation_function" {
-  description = "Transformation function for latency detector (mean, min, max)"
-  type        = string
-  default     = "min"
+variable "latency_timer" {
+  description = "Minimum duration of lasting where conditions are true before raising alert in seconds"
+  type        = number
+  default     = 600
 }
 
-variable "latency_transformation_window" {
-  description = "Transformation window for latency detector (i.e. 5m, 20m, 1h, 1d)"
-  type        = string
-  default     = "5m"
+variable "latency_percentage" {
```

Comment:
What about this ?
```suggestion
variable "latency_timer_tolerance" {
```

Reply:
Also, we may want to have different tolerance for warning & critical thresholds.